WebSharper.JsPDF/Main.fs

```diff
@@ -81,101 +82,1654 @@ module Definition =
+                ]
+            let methodsToApply = [
+                "html" => (T<string> + T<JavaScript.HTMLElement>)?src * (!? HTMLOptions)?options ^-> HTMLWorker
+            ]
+            let typesToExport = [
+                Html2CanvasOptions
+                HTMLWorkerProgress
+                HTMLOptionImageType
+                HTMLOptionImage
+                HTMLFontFaceStyle
+                HTMLFontFaceStretch
+                HTMLFontFaceWeight
+                HTMLFontFaceFormat
+                HTMLFontFaceSource
+                HTMLFontFace
+                HTMLOptions
+                HTMLWorkerType
+                HTMLWorkerOutputImgType
+                HTMLWorker
+            ]
+            
+        let typesToExport = [
+            for t in AddImage.typesToExport do t
+            for t in Annotations.typesToExport do t
+            for t in AcroForm.typesToExport do t
+            for t in Html.typesToExport do t
+            for t in AutoPrint.typesToExport do t
+            for t in Context2d.typesToExport do t
+            for t in Canvas.typesToExport do t
+            for t in Cell.typesToExport do t
+            for t in Outline.typesToExport do t
+            for t in SetLanguage.typesToExport do t
+            for t in ViewerPreferences.typesToExport do t
+        ]
+        let methodsToApply : CodeModel.IClassMember list =
+            [
+                for m in AddImage.methodsToApply do m
+                for m in Arabic.methodsToApply do m
+                for m in Annotations.methodsToApply do m
+                for m in AcroForm.methodsToApply do m
+                for m in Html.methodsToApply do m
+                for m in AutoPrint.methodsToApply do m
+                for m in Context2d.methodsToApply do m
+                for m in Canvas.methodsToApply do m
+                for m in Cell.methodsToApply do m
+                for m in Outline.methodsToApply do m
+                for m in FileLoading.methodsToApply do m
+                for m in SplitTextToSize.methodsToApply do m
+                for m in SVG.methodsToApply do m
+                for m in SetLanguage.methodsToApply do m
+                for m in TotalPages.methodsToApply do m
+                for m in ViewerPreferences.methodsToApply do m
+                for m in VFS.methodsToApply do m
+                for m in XmpMetadata.methodsToApply do m
+                
+            ]
+    
+    JsPDFClass
+    |+> Instance [
+        for m in Plugins.methodsToApply do m
+    ]
+    |+> Static [
+        for m in Plugins.AcroForm.staticToApply do m
+    ]
+    |> ignore
+    
     let Assembly =
         Assembly [
             Namespace "WebSharper.JsPDF.Resources" [
-                Resource "Js" "https://cdnjs.cloudflare.com/ajax/libs/jspdf/1.3.2/jspdf.debug.js"
-                |> AssemblyWide
+                //Resource "Js" "https://cdnjs.cloudflare.com/ajax/libs/jspdf/1.3.2/jspdf.debug.js"
```

Comment:
@GyGerg We can remove the whole WebSharper.JsPDF.Resources namespace, no need to keep it

Reply:
Updated, thanks!